test/util/FetchUtil.ts

```diff
@@ -104,6 +104,25 @@ export async function patchResource(url: string, query: string, exists?: boolean
   return response;
 }
 
+/**
+ * This is specifically for PATCH requests which are expected to succeed.
+ */
+export async function patchResourceN3(url: string, query: string, exists?: boolean): Promise<Response> {
```

Comment:
Would merge with `patchResource` and have it take an extra `type: 'sparql' | 'n3'` parameter.